README.md

```diff
@@ -78,7 +108,10 @@ your computer went to sleep), you can resume with:
 
 `bash resume.sh jupyter`
 
+If you want to modify the partition flag to have a different gpu setup (other than `--partition gpu --gres gpu:1`) then you should set this **entire** string for the partition variable.
```

Comment:
Is this line meant to go in the Parameters subsection you created above?  Might be good to supply an example too.

Reply:
yes :)